hc/front/tests/test_add_shopify.py

```diff
@@ -0,0 +1,92 @@
+from hc.test import BaseTestCase
+from shopify_trois import Credentials, Collection
+from shopify_trois.models import Shop
+from shopify_trois.engines.http import Json as Shopify
+from django.core.urlresolvers import reverse
+import shopify
+import requests
+from hc.api.models import Check
+import os
+
+class AddShopifyAlertTestCase(BaseTestCase):
```

Comment:
A number of these do seem to be deterministic and it looks like they are dependent on actual data from shopify. This can lead to failure in the tests if the data is changed or when the tests are run without an internet connection. Mock out the responses from shopify and use them to refactor your tests.